endpoints/openrtb2/auction_test.go

```diff
@@ -43,6 +44,11 @@ import (
 
 const jsonFileExtension string = ".json"
 
+func TestMain(m *testing.M) {
+	jsoniter.RegisterExtension(&jsonutil.RawMessageExtension{})
+	m.Run()
+}
```

Comment:
I think you need to capture the result of `m.Run()` and use it to fail the process in case of failure. Something like:

```
func TestMain(m *testing.M) {
	jsoniter.RegisterExtension(&jsonutil.RawMessageExtension{})
	os.Exit(m.Run())
}
```

Same for other `TestMain` functions.

Reply:
Added